### DISCUSSION TITLE
When will a version of this be put on nuget that is not pre-release?

### DISCUSSION THREAD

**Discussion by solmead** [SixLabors/ImageSharp.Drawing#78] [General] [answered]
As the header says, I would love to use this, but can't use it in projects because it's marked pre-release. Anyway a version 0.X could be released?

**Comment by JimBobSquarePants** [comment]
When it's ready.

**Comment by antonfirsov** [comment, marked as answer]
That mark is here for a good reason. Now that the main library is finally in 1.0, one of the main focus of the next few months will be to bring ImageSharp.Drawing into a production-ready state.

When it reaches Release Candidate, it will be quite safe to use in production (just like it was with the main library), but we don't plan to remove the pre-release mark before reaching the final 1.0 milestone.

**Reply by solmead** [reply]
Thank you, that makes sense. I've been following this and the main project since December of 2018, and working with the beta versions since then in my .net standard code.
